frontend/src/service/Url.service.ts

```diff
@@ -149,9 +149,14 @@ export class UrlService {
           resolve(res.data.authMutation.createURL);
         })
         .catch(({ graphQLErrors, networkError, message }) => {
-          console.log(graphQLErrors);
+          if (networkError) {
+            return reject([Err.NetworkError]);
+          }
+          if (!graphQLErrors || graphQLErrors.length === 0) { 
+            return reject([Err.Unknown]);
+          }
           const errCodes = graphQLErrors.map(
-            (graphQLError: GraphQlError) => graphQLError.extensions.code
+            (graphQLError: GraphQlError) => (graphQLError.extensions) ? graphQLError.extensions.code : Err.Unknown
```

Comment:
Are parentheses needed for `(graphQLError.extensions)`?

Reply:
Nope they can be left out :)